spinnaker/spinnaker_system/google_http_lb_upsert_scenario.py

```diff
@@ -659,6 +660,7 @@ def delete_server_group(self):
       job=[{
         'cloudProvider': 'gce',
         'serverGroupName': group_name,
+        'asgName': group_name,
```

Comment:
That's not relevant, I'll take it out. I just made it match what's coming from Deck.